### PR TITLE
Docs: Fix links

### PR DESCRIPTION
**Pull request by oddlittlebird**
Fixing some broken links.

### CODE REVIEW

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

File: docs/sources/shared/test.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (wbrowne) [inline docs/sources/shared/test.md:1]: Should this file be committed too? 🤔

Comment (achatterjee-grafana) [reply docs/sources/shared/test.md:1]: @oddlittlebird will merge this soon.

Comment (oddlittlebird) [reply docs/sources/shared/test.md:1]: Yes

**Pull request merged**